elide-integration-tests/src/test/resources/ResourceIT/createChildRelateExisting.json

```diff
@@ -8,16 +8,27 @@
             },
             "relationships":{
                 "friends":{
+                    "links": {
+                        "self": "/parent/7/child/8/relationships/friends",
+                        "related": "/parent/7/child/8/friends"
+                    },
                     "data":[]
                 },
                 "parents":{
+                    "links": {
+                        "self": "/parent/7/child/8/relationships/parents",
+                        "related": "/parent/7/child/8/parents"
```

Comment:
Here is an example of the above comment.  This should be /parent/7/children/8/parents